pkg/runner/runner.go

```diff
@@ -74,18 +75,28 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	flag.StringVar(&migrateOption, "migrate", "true", "Update the Prometheus SQL schema to the latest version. Valid options are: [true, false, only].")
 	flag.BoolVar(&cfg.UseVersionLease, "use-schema-version-lease", true, "Use schema version lease to prevent race conditions during migration.")
 	flag.BoolVar(&cfg.InstallTimescaleDB, "install-timescaledb", true, "Install or update TimescaleDB extension.")
-	flag.BoolVar(&cfg.ReadOnly, "read-only", false, "Read-only mode for the connector. Operations related to writing or updating the database are disallowed. It is used when pointing the connector to a TimescaleDB read replica.")
-	flag.BoolVar(&cfg.AdminAPIEnabled, "web-enable-admin-api", false, "Allow operations via API that are for advanced users. Currently, these operations are limited to deletion of series.")
+	flag.StringVar(&cfg.TLSCertFile, "tls-cert-file", "", "TLS Certificate file for web server, leave blank to disable TLS.")
+	flag.StringVar(&cfg.TLSKeyFile, "tls-key-file", "", "TLS Key file for web server, leave blank to disable TLS.")
 	envy.Parse("TS_PROM")
 	// Ignore errors; CommandLine is set for ExitOnError.
 	_ = flag.CommandLine.Parse(args)
 
+	// Checking if TLS files are not both set or both empty.
+	if (cfg.TLSCertFile != "") != (cfg.TLSKeyFile != "") {
```

Comment:
shouldn't we use `len()` here instead of `""`?

Reply:
Why `len`? We are checking against the default value here.

Reply:
i think if we want to check if the string is empty or not, i think we should do with `len()`. I recall reading source code of other big projects and thats how they did when things related to checking empty strings came into picture. So thought, we could do the same here.

 Also, it makes the code look cleaner IMO.

```suggestion
	if len(cfg.TLSCertFile) > 0 != len(cfg.TLSKeyFile) > 0 {
```

Reply:
I don't think your suggestion looks cleaner. In fact, looks harder to read. I need a better reason to change this.